eng/pipelines/templates/steps/apiview-ui-tests.yml

```diff
@@ -92,7 +92,7 @@ steps:
     "FIXTURE_DIR": "$(Build.BinariesDirectory)"
     "APIVIEW_API_KEY": "$(azuresdk-apiview-apikey)"
 
-- task: PublishBuildArtifacts@1
+- task: 1ES.PublishPipelineArtifact@1
   inputs:
-    pathtoPublish: '$(Build.SourcesDirectory)\src\dotnet\APIView\APIViewWeb\Client\playwright-report'
-    artifactName: 'Client-Side Test Reports'
\ No newline at end of file
+    path: '$(Build.SourcesDirectory)\src\dotnet\APIView\APIViewWeb\Client\playwright-report'
+    artifact: 'Client-Side Test Reports'
```

Comment:
is this the report of the tests? could you point me to where I could find it?

Reply:
I don't yet have reporting for the tests. But it reports on the pipeline log and will break if a test fails.

Reply:
great. if we can see which test breaks then we should be good